File: chap09/run-model9-2.R
library(cmdstanr)

d <- read.csv(file='input/data-tortoise-hare.csv')
data <- list(N=2, G=nrow(d), g2L=d$Loser, g2W=d$Winner)

model <- cmdstan_model(stan_file='model/model9-2.stan')
fit <- model$sample(data=data, seed=123, parallel_chains=4)
fit$save_object(file='output/result-model9-2.RDS')

File: chap09/run-model9-2.py
import pandas
import cmdstanpy

d = pandas.read_csv('input/data-tortoise-hare.csv')
data = {'N':2, 'G':len(d), 'g2L':d.Loser, 'g2W':d.Winner}

model = cmdstanpy.CmdStanModel(stan_file='model/model9-2.stan')
fit = model.sample(data=data, seed=123, parallel_chains=4)
fit.save_csvfiles('output/result-model9-2')
